.gitignore

```diff
@@ -2,7 +2,7 @@
 
 /api/env
 /api/__pycache__
-
+/api/app/__pycache__
```

Comment:
Since we're moving forward with Firebase, you should delete the `/api/app/` directory.

Reply:
done